src/entities/GroceryItem.ts

```diff
@@ -0,0 +1,70 @@
+import {
+	Entity,
+	PrimaryGeneratedColumn,
+	Column,
+	BaseEntity,
+	CreateDateColumn,
+	ManyToOne,
+} from 'typeorm';
+import { ObjectType, Field } from 'type-graphql';
+import { GroceryList } from './GroceryList';
+import { GroceryItemInfo } from './GroceryItemInfo';
+@ObjectType()
+@Entity('groceryItems')
+export class GroceryItem extends BaseEntity {
+	// ---------------- fields ----------------
+	@Field()
+	@PrimaryGeneratedColumn({ type: 'int' })
+	id!: number;
+
+	@Field(() => Number)
+	@Column({ type: 'int' })
+	groceryItemInfoId!: number; // foreign key to Fridge table
```

Comment:
Maybe we don't need a groceryItemInfo table. We can merge the groceryItemInfo table and the groceryItem table together.